### PR TITLE
parameters for learning rate reduction upon loss plateau mutable by CLI

### PR DESCRIPTION
**Pull request by mooratov**
the default parameters for ReduceLROnPlateau are fairly aggressive. Probably fine for most cases, but can be a problem if number of steps per epoch is low or validation set is limited.

### CODE REVIEW

**Comment by hgaiser** [comment]
LGTM, thanks :)
